nrc_ws/src/nrc_nav/src/nrc_drive_pp.py

```diff
@@ -16,11 +16,15 @@
 instructions = None
 # pure pursuit path
 pp = PurePursuit()
+pp.add_point(0, -1)
+pp.add_point(2, -1)
+pp.add_point(2, 1)
+pp.add_point(0, 1)
```

Comment:
Hard-coded points for the indoor test. Probably fine to push to master since this robot won't see competition anyways 😢